# joint description values
# JointType type
int32 type
float32 dh_theta
float32 dh_alpha
float32 dh_d
float32 dh_a
float32 dh_min
float32 dh_max
